README.md

```diff
@@ -85,3 +85,41 @@ Do the following:
    * `go get github.com/some/lib@v1.2.3`: Get a particular version.
 2. Run `go mod tidy` to remove any unused modules.
 3. Run `go mod verify` to add packages needed for test packages etc.
+
+## Pulling the docker image
+
+Unfortunately the Github packages docker repo is not very "public", more on this can be found in this community [thread](https://github.community/t5/GitHub-Actions/docker-pull-from-public-GitHub-Package-Registry-fail-with-quot/td-p/32782/page/4). This means a docker login needs to be done before the images can be pulled. To do this follow these steps:
```

Comment:
Change to "GitHub" throughout the text (where applicable)?
Change ..."a docker login needs to be done"... to ..."a Docker login is needed before..."?

Reply:
done